src/components/Toggle/tests/Toggle.test.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import {noop} from '@shopify/javascript-utilities/other';
+import {mountWithAppProvider} from 'test-utilities';
+import Labelled from '../../Labelled';
+import TextContainer from '../../TextContainer';
+import Toggle from '../Toggle';
+
+describe('<Toggle />', () => {
+  const mockProps = {
+    label: 'label',
+    onChange: noop,
+  };
+
+  describe('accessibilityLabel', () => {
```

Comment:
You might want to test all accessibility props in this `describe` (ie. aria-checked and  aria-disabled)

Reply:
@mehdi-loup Did you have any thoughts about the change in background color?

Reply:
You mean the transition for the bg color? I think it is great you added this.

For the colours themselves, if that is your question, I remember that Fred Audet validated them.